src/Kinepolis/AuthenticatedKinepolisClient.php

```diff
@@ -72,6 +72,34 @@ public function getMovieDetail(string $token, int $mid): array
         return $contents['movies'][0];
     }
 
+    public function getTheaters(string $token): array
+    {
+        $request = new Request(
+            'GET',
+            $this->movieApiBaseUrl . 'services/content/1.1/theaters',
+            $this->createHeaders($token)
+        );
+
+        $response = $this->client->sendRequest($request)->getBody()->getContents();
+        $contents = Json::decodeAssociatively($response);
+
+        return $contents['theatres'];
```

Comment:
Should we do a check here if theatres exist?

```suggestion
        return $contents['theatres'] ?? [];
```

In general I am missing a check if the API request failed? How would that error be capture if for example the API is down?

Reply:
If the entire KinepolisAPI is down, we could not fetch a `token` and thus not continue.
In most cases, including this one, I am in favour of failing hard & fast.
If we cannot fetch the theaters, we cannot fetch the Prices, so we could not add `PriceInfo`.

Reply:
There might be a situation where the API call succeeds, but returns something unexpected.
I am fine with throwing an exception in such case, but do think we should test for it.
Maybe add that catched exception to a log?

Reply:
I agree, any exception thrown by the `Client` should be caught in the `KinepolisService` & written to the log+sentry.